youtube_dl/extractor/orf.py

```diff
@@ -376,6 +376,38 @@ class ORFOE1IE(ORFRadioIE):
     }
 
 
+class ORFRadiothekIE(ORFRadioIE):
+    _VALID_URL = r'https?://radiothek\.orf\.at/(?P<station>\w+)/(?P<date>[0-9]+)/(?P<show>\w+)'
```

Comment:
This can be moved to station extractor regexes eliminating this extractor at all.